VegProcessor/configs/hsi_config_2ft_slr_dry.yaml

```diff
@@ -0,0 +1,38 @@
+# config.yaml
+
+raster_data:
+  dem_path: "/Users/dillonragar/data/cpra/AMP_NGOM2_DEM_60m_Bilinear_Resample.tif" # str
+  #wse_directory_path: "/Users/dillonragar/data/tmp/cpra/AMP_SimulationResults" # must be unzipped
+  wse_directory_path: "/Users/dillonragar/data/tmp/cpra/2ft-slr-sequence-dry"
+  veg_base_raster: "/Users/dillonragar/data/cpra/AMP_10km_LULC_60m_Nearest_Resample/AMP_10kmBuffer_LULC_60m_Nearest_Resample.tif" #str
+  veg_keys: "/Users/dillonragar/data/cpra/AMP_10km_LULC_60m_Nearest_Resample/AMP_10kmBuffer_LULC_60m_Nearest_Resample.tif.vat.dbf"
+  wpu_path: /Users/dillonragar/Library/CloudStorage/OneDrive-LynkerTechnologies/2024 CPRA Atchafalaya DSS/data/WPU_id_60m.tif # 60m domain with pixels labeled by WPU
+  salinity_raster: #str (optional)
+  veg_type_path: "/Users/dillonragar/data/cpra/AMP_VEG_S05_G504_AB_O_ANN_01_25/AMP_VEG_S05_G504_AB_O_ANN_01_25_DATA.nc" # VegTransition output folder
+  flotant_marsh_raster: "/Users/dillonragar/data/cpra/AMP_lndtyp_60m_Nearest_Resample/AMP_lndtyp_60m_Nearest_Resample.tif"
+  #flotant_marsh_keys: "/Users/jessicagarrett/Documents/CPRA Royal 2024/GIS/AMP_lndtyp_60m_Nearest_Resample/AMP_lndtyp_60m_Nearest_Resample.tif.vat.dbf"
+
+simulation:
+  water_year_start: 1951 # int
+  water_year_end: 1975 # int
+  wse_sequence_input: True # bool if WSE input is synthetic
+  run_hsi: True
+
+metadata:
+  model: HSI
+  scenario: "05" # scenario info for plots and metadata
+  group: G500
```

Comment:
@dmragar - Is it worth changing these to version "G504"?  Just for compliance with veg transitions and our current status of `main`.

Reply:
This is a good point. Maybe it is best to keep group numbers out of main? (Then it is a good reminder to create a branch for results that will go to Royal).

Reply:
Up to you, really. Right now `main` is the most "up to date version G504" so I think it's fine to include the group # here.  Fine with it either way...